src/OpenTelemetry.AutoInstrumentation/Instrumentations/RabbitMq6/Integrations/AsyncDefaultBasicConsumerIntegration.cs

```diff
@@ -10,19 +10,20 @@
 namespace OpenTelemetry.AutoInstrumentation.Instrumentations.RabbitMq6.Integrations;
 
 /// <summary>
-/// RabbitMq AsyncEventingBasicConsumer.HandleBasicDeliver integration.
+/// RabbitMq AsyncDefaultBasicConsumer.HandleBasicDeliver integration.
 /// </summary>
 [InstrumentMethod(
     assemblyName: IntegrationConstants.RabbitMqAssemblyName,
-    typeName: IntegrationConstants.AsyncEventingBasicConsumerTypeName,
+    typeName: IntegrationConstants.AsyncDefaultBasicConsumerTypeName,
     methodName: IntegrationConstants.HandleBasicDeliverMethodName,
     returnTypeName: ClrNames.Task,
     parameterTypeNames: new[] { ClrNames.String, ClrNames.UInt64, ClrNames.Bool, ClrNames.String, ClrNames.String, IntegrationConstants.BasicPropertiesInterfaceTypeName, $"System.ReadOnlyMemory`1[{ClrNames.Byte}]" },
     minimumVersion: IntegrationConstants.MinSupportedVersion,
     maximumVersion: IntegrationConstants.MaxSupportedVersion,
     integrationName: IntegrationConstants.RabbitMqByteCodeIntegrationName,
-    type: InstrumentationType.Trace)]
-public static class AsyncEventingBasicConsumerIntegration
+    type: InstrumentationType.Trace,
+    integrationKind: IntegrationKind.Derived)]
```

Comment:
I think we also need to declare an instrumentation for this class that is _not_ derived, as Derived would only instrument base classes. Can you please validate in the integration test that we can also instrument a consumer of type `RabbitMQ.Client.AsyncDefaultBasicConsumer`?